Composer/packages/integration-tests/scripts/e2e.js

```diff
@@ -61,17 +60,18 @@ Wait for the server to come up and then start cypress.
 
 async function setup() {
   try {
-    await mkdir(process.env.COMPOSER_BOTS_FOLDER);
+    await fs.ensureDir(process.env.COMPOSER_BOTS_FOLDER);
   } catch (err) {
     process.stderr.write('There was a problem setting up.\n');
     process.stderr.write(`Error:\n${err.message}\n`);
   }
 }
 
 async function run() {
+  console.log('COMPOSER ROOT', composerRootDir);
   return new Promise((resolve) => {
     const startCommand = isDev ? 'start:dev' : 'start';
-    const server = spawn('yarn', [startCommand], { cwd: path.resolve(rootDir), stdio: 'inherit' });
+    const server = spawn('yarn', [startCommand], { cwd: composerRootDir, stdio: 'inherit' });
```

Comment:
While you are touching this file, can you add the following? This script doesn't work on Windows.

```
const spawnOptions =  { cwd: composerRootDir, stdio: 'inherit' };
if (process.platform === 'win32') {
  spawnOptions.shell = true;
}
const server = spawn('yarn', [startCommand], spawnOptions);
```


I have a branch that has this change but never submitted a PR.

Reply:
Will do. Thanks